configs/eln-repo-split.yaml

```diff
@@ -0,0 +1,168 @@
+document: eln-repo-split
+version: 1
+data:
+
+  name: Initial content split
+  maintainer: jwboyer
+
+  baseos:
+    must:
+    - kernel
+    - grub2
+    - glibc
+    - bash
+    - rpm
+    - dnf
+    - kmod
+    - linux-firmware
+    - curl
+    - openssh-clients
+    - openssh-server
+    - policycoreutils
+    - selinux-policy
+    - selinux-policy-targeted
+    - systemd
+
+  appstream:
+    must:
```

Comment:
@jwboyer are you ok with changing this to `want`? The appstream list doesn't have a `must` as it's the default list, and it could result in conflicting or breaking results if something pulled to baseos would require something that would need to be suddenly forced in appstream.

Reply:
Ah.  Yes.  Do you want me to change and resubmit?

Reply:
I rebased and fixed this.  I have no idea if that's what I was supposed to do, but I did! :smile: